shared/js/template.js

```diff
@@ -114,24 +114,25 @@
    *                          HTML that is known and are
    *                          "known" to ignore.
    */
-  Template.prototype.interpolate = function(data, options) {
-    // This _should_ be rewritten to use Firefox's support for ES6
-    // default parameters:
-    // ... = function(data, options = { safe: [] }) {
-    //
-    options = options || {};
-    options.safe = options.safe || [];
```

Comment:
If we want to prevent unhandled errors related to the safe property we may replace its auto-initialization by this :
`options.safe = Array.isArray(options.safe) ? options.safe : [];`

or even creating a variable to use in the ternary operator :
```javascript
  Template.prototype.interpolate = function(data = {}, options = {}) {
    var hasSafeProperties = Array.isArray(options.safe) && options.safe.length;

    return this.extract().replace(rmatcher, function(match, property) {
      property = property.trim();

      var isPropertySafe = hasSafeProperties && options.safe.includes(property);
// [...]
      return isPropertySafe ? unsafeStr : Template.escape(unsafeStr);
    });
  };
```

what do you think ?